src/base/bittorrent/session.h

```diff
@@ -70,6 +70,13 @@ enum MaxRatioAction
     EnableSuperSeeding = 2
 };
 
+enum TLSVersion
+{
+    TLS11 = 0,
+    TLS12 = 1,
+    TLS13 = 2
```

Comment:
I don't think you need to assign values to it explicitly.

Reply:
why do you duplicate this enum from libtorrent? Is it because you need to guarantee that the integer values assigned to them are stable, because it's part of your settings file format? If so, it probably *does* make sense to assign those values explicitly. If one option is removed in the future, you would still want the existing enums to keep their value.